README.md

```diff
@@ -2,8 +2,8 @@
 
 This [GitHub Action][github actions] publishes to npm with the following conventions:
 
-1. If we're on the `master` branch, the `version` field is used as-is and we just run `npm publish --access public`.
-   - After publishing a new version on the `master` branch, we tag the commit SHA with `v{version}` via the GitHub API.
+1. If we're on the default branch, the `version` field is used as-is and we just run `npm publish --access public`.
+   - After publishing a new version on the default branch, we tag the commit SHA with `v{version}` via the GitHub API.
```

Comment:
I wonder if using the term "release branch" would be better here since that's the name of the setting. What do you think?

Reply:
true.. maybe the setting should be called `default branch` though 🤔 `release branch` sounds more like the branch that you put together before you do a release, the one that you eventually merge into your default branch

Reply:
I ended up renaming this to `default` branch because I think it's confusing to use `release-branch` here when GitHub refers to the "main" branch as the default branch and there's _other_ behavior tied to an actual "release branch" (shipping canary releases)